Controllers/OrderController.js

```diff
@@ -28,4 +28,22 @@ let createNewOrder = (req, res) => {
             })
         }
     })
-}
\ No newline at end of file
+}
+
+const addToOrder = (req, res) => {
+    DbService.connectToDb(async (db) => {
+        try {
+            await OrderService.addOneItemToOrder(db, req)
+            let response = JSONResponseService.generateSuccessResponse()
+            response.message = "Dish successfully added to order"
+            return res.json(response)
+        } catch (e) {
+            let response = JSONResponseService.generateFailureResponse()
+            response.message = "Dish not found so cannot add to order"
+            return res.json(response)
+        }
+    })
```

Comment:
do you want to have an external error after that leads to db failure?

Reply:
No, we have taken db failure into account. If db fails there will be no JSON response at all and a default error message :)